detox/.detoxrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "testRunner": "jest",
+  "runnerConfig": "e2e/config.json",
+  "configurations": {
+    "ios.sim.debug": {
+      "binaryPath": "../ios/Build/Products/Debug-iphonesimulator/Mattermost.app",
+      "type": "ios.simulator",
+      "device": {
+        "type": "iPhone 11"
+      }
+    },
+    "ios.sim.release": {
+      "type": "ios.simulator",
+      "binaryPath": "../ios/Build/Products/Release-iphonesimulator/Mattermost.app",
+      "build": "make ios-sim-x86_64",
+      "device": {
+        "type": "iPhone 11"
+      }
+    },
+    "android.emu.debug": {
+      "type": "android.emulator",
+      "binaryPath": "../android/app/build/outputs/apk/debug/app-debug.apk",
+      "build": "cd ../android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ../detox",
+      "device": {
+        "avdName": "detox_emu_api_29"
+      }
+    },
+    "android.emu.release": {
+      "type": "android.emulator",
+      "binaryPath": "../android/app/build/outputs/apk/release/app-release.apk",
+      "build": "cd ../android && ./gradlew assembleRelease assembleAndroidTest -DtestBuildType=release && cd ../detox",
```

Comment:
This is a quick build only. Permanent fix is to move into fastlane. With that, before doing detox build, either run `make run-android` or `make build-android` to setup/download/compile necessary library/packages.